third_party/move/move-vm/types/Cargo.toml

```diff
@@ -18,11 +19,11 @@ smallbitvec = "2.5.1"
 smallvec = "1.6.1"
 triomphe = "0.1.9"
 
-bcs = { workspace = true }
 move-binary-format = { path = "../../move-binary-format" }
-
 move-core-types = { path = "../../move-core/types" }
 
+bcs = { workspace = true }
```

Comment:
what's up with the strange ordering here? is it because some have versions, some have path, etc?

Reply:
I don't think Move repo enforces any sort of ordering, seems random to me. Re-ordered to have all external dependencies first, and then have the move ones.

Maybe @vgao1996 or @runtian-zhou know?